styles.css

```diff
@@ -35,8 +37,14 @@ header {
 }
 
 .instructions {
-    color: #500110;
+    color: #464444;
     margin-bottom: 10px;
```

Comment:
Perhaps try to use 'rem' for all margins and paddings, so the code looks more consistent. This will probably count in code interviews etc.